content/docs/operating/configuration.md

```diff
@@ -83,6 +83,15 @@ dynamically discovered using one of the supported service-discovery mechanisms.
 Additionally, `relabel_configs` allow advanced modifications to any
 target and its labels before scraping.
 
+If the target requires authentication, the following options are available:
+
+* `basic_auth`
```

Comment:
I know this list will be self-explaining to many, but maybe a brief description after each point?